txn_writer.go

```diff
@@ -152,7 +152,7 @@ func (sw safeWriter) writeAllSafe(forceVendor bool) error {
 
 	if writeM {
 		if _, err := os.Stat(mpath); err == nil {
-			// move out the old one
```

Comment:
Those examples are not proper sentences but mostly phrases or adjectives that is representing the decision.

```
// tentatively okay - read to make sure
// comma-separated list
```
vs

```
// It is tentatively ok to accept these extensions, the source should be read to determine.
// Accept a comma-separated list.
```

If you are writing a proper sentence, start with upper case and end with a full stop. It is seriously nothing but basic language conventions.